src/handlers/graphql.ts

```diff
@@ -15,9 +13,8 @@ export interface MyContext extends BaseContext {
 export const apolloServer = new ApolloServer<MyContext>({
 	typeDefs,
 	resolvers,
-	cache: new KeyvAdapter(redisClient),
 	persistedQueries: {
-		ttl: Number(REDIS_CACHE_TTL_SECONDS),
+		ttl: Number(CACHE_TTL_SECONDS),
```

Comment:
You can remove this whole section tbh. We're not going to have persistedQueries in this example.